docs/conf.py

```diff
@@ -14,14 +14,14 @@
 from pathlib import Path
 
 current_dir = Path(__file__).parents[1]
-code_directory = current_dir / "maykin_django_utils"
+code_directory = current_dir / "django_common"
 
 sys.path.insert(0, str(code_directory))
 
 
 # -- Project information -----------------------------------------------------
 
-project = "maykin_django_utils"
+project = "maykin-common"
```

Comment:
```suggestion
project = "maykin-django-common"
```
the naming is a bit awkward, but they all come down to the same idea:

* github: maykinmedia/django-common
* RTD: maykin-django-common
* docs: maykin-django-common
* pypi: maykin-common

the `maykinmedia` / `maykin` prefix makes it clear that this is just for Maykin and not the broader django/python community (they can of course use it, but we won't maintain/keep stuff that we don't use ourselves)

The `django` part makes it clear it's for the backend, to differentiate it from `maykinmedia/client-common`, for example. In the Python package, we omit it because we're a django shop so it's just extra letters to type.